res/layout/backup_provider_fragment.xml

```diff
@@ -108,4 +108,17 @@
 		android:layout_marginBottom="16dp"
 		android:contentDescription="@string/qrscan_title" />
 
+	<TextView
+		android:id="@+id/bottom_text"
```

Comment:
maybe also needs to be centered horizontally with `android:gravity="center_horizontal"` or so

Reply:
added a commit